about.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Make-up blog</title>
+</head>
+<body>
+    <header>
+        <div class="first-navbar">
+            <a href="https://www.facebook.com"> <img src="./images/facebook_icon.png"> </a>
+            <a href="https://www.facebook.com"> <img src="./images/twitter_icon.png"> </a>
+        </div>
+        <nav>
+            <div class="ham-cross">
+                <span class="hamburger-icon1">&#9776</span>
+                    <span class="cross hidden">X</span>
+            </div>
+            <div class="logo-items">
+                <span class="nav-logo"> <a href="index.html"> <img src="./images/logo.jpg" alt=""  class="header-logo" srcset=""></a></span>
+                    <ul class="second-navbar">
+                        <li class="second-navbars-links"><a href="about.html">About</a></li>
```

Comment:
- [ ] While the user is at the About page, this link should link back to the Home page. Kindly update this.